Makefile

```diff
@@ -99,6 +100,8 @@ define install_acrn_debug
 endef
 
 HV_MAKEOPTS := -C $(T)/hypervisor BOARD=$(BOARD) SCENARIO=$(SCENARIO) HV_OBJDIR=$(HV_OUT) RELEASE=$(RELEASE)
+board_inspector:
+	@python3 hypervisor/scripts/gen_acrn_deb.py $(ROOT_OUT) $(FULL_VERSION) board_inspect
```

Comment:
```suggestion
	@python3 hypervisor/scripts/gen_acrn_deb.py $(ROOT_OUT) $(FULL_VERSION) board_inspector
```

Reply:
Is `gen_acrn_deb.py` designed to be able to generate all deb packages (including DM and tools)? If so, it does not look wise to put it under `hypervisor/scripts`.